joblib/test/data/generate_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+for i in py26-np16 py27-np17 py33-np18 py34-np19 py35-np19
```

Comment:
A bit specific to your environment. It may be worth adding something like this since it could be useful for regenerating pickles. On the fence on this one. At the very least do something like

```sh

# Set this variable to match the list of conda/virtualenv environments you want to regenerate the pickles for
ENVIRONMENTS="py26-np16 ..."

for env in $ENVIRONMENTS
do
```